### PR TITLE
New package: NIS.One-Click-Backup version 1.2

### PR DESCRIPTION
**Pull request by dev-nis**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119260)

### CODE REVIEW

**Comment by dev-nis** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)  ---
> System.ComponentModel.Win32Exception (216): The specified executable is not a valid application for this OS platform. 
--- End of stack trace from previous location where exception was thrown ---    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)

File: manifests/n/NIS/One-Click-Backup/1.2/NIS.One-Click-Backup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: NIS.One-Click-Backup
+PackageVersion: "1.2"
+InstallerType: nullsoft
+Installers:
+- InstallerUrl: https://gitlab.com/dev-nis/nis-one-click-backup-qt/uploads/ded806b331306a9dfef4eb388cae135b/nis-ocb-install.exe
+  Architecture: x86
```
Comment (stephengillie) [inline manifests/n/NIS/One-Click-Backup/1.2/NIS.One-Click-Backup.installer.yaml:9]:
```suggestion
  Architecture: x64
```

**Comment by dev-nis** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
I believe it passed validation, but I'm going to run it again - without the label - to make sure.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 12:52:59 PM:  > 2023-09-12 12:52:58.366 [FAIL] Installer failed security check. Url: https://gitlab.com/dev-nis/nis-one-click-backup-qt/uploads/ded806b331306a9dfef4eb388cae135b/nis-ocb-install.exe Result: 0x80004005 

This might be a false positive, and we can retry tomorrow.

**Comment by dev-nis** [comment]
> Installer failed security check.

@stephengillie what does this exactly mean? What type of security is checked here?

**Comment by stephengillie** [comment]
> > Installer failed security check.
> 
> @stephengillie what does this exactly mean? What type of security is checked here?

Hi @dev-nis,

It's like a virus scan. We scan packages several times during different parts of validation.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**